java/registry/src/main/java/io/opensaber/registry/app/OpenSaberApplication.java

```diff
@@ -16,19 +16,19 @@
 @ComponentScan(basePackages = { "io.opensaber.registry", "io.opensaber.pojos"})
 public class OpenSaberApplication {
 	private static ApplicationContext context;
+	private static SpringApplication application = new SpringApplication(OpenSaberApplication.class);
 
 	public static void main(String[] args) {
-		SpringApplication application = new SpringApplication(OpenSaberApplication.class);
-		if(args.length != 0 && args[0].equals(WebApplicationType.NONE.toString())){
-			application.setWebApplicationType(WebApplicationType.NONE);
-		} else {
-			application.setWebApplicationType(WebApplicationType.SERVLET);
-		}
-
+		application.setWebApplicationType(WebApplicationType.SERVLET);
```

Comment:
why is this still existing? This isn't doing much value as this is the default.

Reply:
Value is nothing here, but i thought it will be good if it has WebApplicationType.SERVLET, since for normal application-context we added WebApplicationType.NONE.

Reply:
Now it is removed.